examples/web-components-kitchen-sink/.storybook/preview.js

```diff
@@ -1,10 +1,31 @@
+/* eslint-disable */
 /* global window */
 
-import { addParameters, setCustomElements } from '@storybook/web-components';
+import {
+  addParameters,
+
+  // setCustomElementsManifest,
+  // ☝️☝️☝️☝️☝️☝️
+  // @TODO: cant import this function for some reason after running `yarn bootstrap --core`
```

Comment:
Hi @shilman , still havent managed to figure this out unfortunately ☹️ 

I have:
- Made my changes locally to: `addons/docs/src/frameworks/web-components/custom-elements.ts` and `app/web-components/src/client/customElements.ts`
- run `yarn bootstrap --core`
- run `cd examples/web-components-kitchen-sink/`
- run `yarn storybook`

But for some reason its giving me this error:
![image](https://user-images.githubusercontent.com/17054057/122404852-dc5fe380-cf7f-11eb-8b2f-5f6ba40fa506.png)

I have been able to test my changes by just copy pasting the function into `preview.js` and trying it out, but (if I understand correctly) I'd expect the function to be exposed from `@storybook/webcomponents` correctly after running `yarn bootstrap --core`.

Would you mind taking a look?